Bottom: 2a48d320de7c438e40d0bb2e8b080e61f78f7922
Top:    3a40661e5f8d2b112707dba8ccea8740d28b1be2
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2013-01-01 19:30:38 +0000

find.c: Remove some unnecessary includes.


---

diff --git a/src/find.c b/src/find.c
index c3bf707..0f85f3e 100644
--- a/src/find.c
+++ b/src/find.c
@@ -65,26 +65,15 @@
 #include "config.h"
 #endif
 
-#include <stdlib.h>
-#ifdef HAVE_STRING_H
-#include <string.h>
-#endif
-#include <math.h>
 #include <setjmp.h>
 #include <assert.h>
 
-#ifdef HAVE_SYS_TIMES_H
-#include <sys/times.h>
-#endif
-
 #include "global.h"
 
-#include "crosshair.h"
 #include "data.h"
 #include "draw.h"
 #include "error.h"
 #include "find.h"
-#include "mymem.h"
 #include "misc.h"
 #include "rtree.h"
 #include "polygon.h"
